bootloader/src/pyi_utils.c

```diff
@@ -740,6 +740,19 @@ pyi_utils_set_environment(const ARCHIVE_STATUS *status)
     return rc;
 }
 
+int
+set_systemd_env(pid_t pid)
+{
```

Comment:
Please add a comment, what this does and why. Basically this si what you've written in the commit message. E.g.

> if the program is actived by a systemd socket, LISTEN_PID ist set and needs to be changed to the child pid of the bootloader